app/controllers/leidinggevendenbeheer/bestuursfunctie/contact-info.js

```diff
@@ -0,0 +1,17 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class LeidinggevendenbeheerBestuursfunctieContactInfoController extends Controller {
+  @service router;
+
+  // TODO before a dialog box was shown when leaving the form with inserted data without saving.
```

Comment:
sounds like a good idea! please make a story for it in the backlog!

Reply:
done